sites/skeleton.dev/src/routes/(inner)/utilities/table-of-contents/+page.svelte

```diff
@@ -0,0 +1,161 @@
+				<p>Set the <code class="code">tocCrawler</code> action on the element with headings to catalog. Supports H2-H6 by default.</p>
+				<CodeBlock
+					language="html"
+					code={`
+<div use:tocCrawler={{ mode: 'generate' }}>
+	<h2>Heading 2</h2>
+	<p>...</p>
+	<h3>Heading 3</h3>
+	<p>...</p>
+</div>
+				`}
+				/>
+				<p>Implement the Table of Contents component, which displays the generated list of links.</p>
+				<CodeBlock language="html" code={`<TableOfContents />`} />
+			</svelte:fragment>
+		</DocsPreview>
+	</svelte:fragment>
+
+	<!-- Slot: Usage -->
+	<svelte:fragment slot="usage">
+		<!-- How it Works -->
+		<section class="space-y-4">
+			<h2 class="h2">How It Works</h2>
+			<!-- Heading IDs -->
+			<h3 class="h3">Heading IDs</h3>
+			<p>Each page heading requires a unique ID that acts as the scroll target for inner-page navigation.</p>
+			<CodeBlock language="html" code={`<h2 class="h2" id="how-it-works">How it Works</h2>`} />
+			<!-- Anchor Links -->
+			<h3 class="h3">Anchor Links</h3>
+			<p>
+				Each link within the Table of Contents then points to the matching target ID as shown below. Note the use of <code class="code"
+					>#</code
+				>. When clicked, the browser will automatically scroll so this element to be at the top of the visible screen.
+			</p>
+			<CodeBlock language="html" code={`<a href="#how-it-works">How It Works</a>`} />
+		</section>
+		<hr />
+		<!-- Settings -->
+		<section class="space-y-4">
+			<h2 class="h2">Settings</h2>
+			<!-- Automatic IDs -->
+			<h3 class="h3">Automatic IDs</h3>
+			<p>
+				Set <code class="code">mode: generate</code> to enable <code class="code">tocCrawler</code> to automatically generate and set unique
+				IDs per all headings within the region the action is applied to.
+			</p>
+			<CodeBlock language="html" code={`<div use:tocCrawler={{ mode: 'generate' }}>`} />
+			<p>See the example below. Note this <u>will not</u> overwrite IDs you have set manually.</p>
+			<CodeBlock
+				language="html"
+				code={`
+<!-- Before: -->
+<h2 class="h2">Title One</h2>
+<h2 class="h2" id="my-custom-id">Title Two</h2>\n
+<!-- After: -->
+<h2 class="h2" id="title-one">Title One</h2>
+<h2 class="h2" id="my-custom-id">Title Two</h2>
+			`}
+			/>
+			<!-- Ignore Headings -->
+			<h3 class="h3">Ignore Headings</h3>
+			<p>
+				To ignore a heading in the target region, append a <code class="code">data-toc-ignore</code> attribute. The crawler will skip this.
+			</p>
+			<CodeBlock language="html" code={`<h2 class="h2" data-toc-ignore>Ignore Me</h2>`} />
+			<!-- Append Headings -->
+			<h3 class="h3">Append Headings</h3>
+			<!-- prettier-ignore -->
+			<p>Use the Tailwind-provided <a class="anchor" href="https://tailwindcss.com/docs/screen-readers#screen-reader-only-elements" target="_blank" rel="noreferrer">Screen Reader</a> <code class="code">.sr-only</code> class to append an invisible heading.</p>
+			<CodeBlock language="html" code={`<h2 class="sr-only">Include Me!</h2>`} />
```

Comment:
I'm a bit confused about the purpose of this section. Is this only for adding invisible headings for screen readers? If so, perhaps we should call it `Adding Invisible Headings` or `Adding Headings for Screen Readers`?